app/src/main/res/layout/feed_pref_skip_dialog.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical" >
+
+    <TextView
+        android:id="@+id/labelSkipIntro"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="24dp"
+        android:layout_marginRight="24dp"
+        android:text="@string/pref_feed_skip_intro" />
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file could need some code cleanup. There are some duplicated namespace declarations, indentation is not consistent and the number of empty lines between Views is not consistent. `Ctrl+Alt+Shift+L` in Android Studio could help.

Reply:
done